perf/json-results.edn

```diff
@@ -0,0 +1 @@
+{:json {"10b" {"r-m-f (defaults)" 22685, "r-m-f (tuned)" 18190, "ring-json" 14263, "muuntaja" 6217}, "100b" {"r-m-f (defaults)" 45163, "r-m-f (tuned)" 21835, "ring-json" 17040, "muuntaja" 9260}, "1k" {"r-m-f (defaults)" 302810, "r-m-f (tuned)" 37115, "ring-json" 32946, "muuntaja" 22497}, "10k" {"r-m-f (defaults)" 2337390, "r-m-f (tuned)" 239191, "ring-json" 267856, "muuntaja" 212611}, "100k" {"r-m-f (defaults)" 7970264, "r-m-f (tuned)" 2315720, "ring-json" 2567301, "muuntaja" 2156446}}}
```

Comment:
Anything goes, I guess. Although automatically incrementing would be better than manual! :)